ptype/inference.py

```diff
@@ -232,29 +246,17 @@ def save_data(dataset, out_path, date, model, forecast_hour):
         None
     """
     dir_str = date.strftime("%Y%m%d")
-    date_str = date.strftime(f"%Y%m%d_{forecast_hour:02}00")
-    file_str = f"ptype_predictions_{model}_{date_str}.nc"
-    full_path = os.path.join(out_path, model, dir_str, file_str)
-    dataset.to_netcdf(full_path)
+    model_run_str = date.strftime("%H%M")
+    os.makedirs(os.path.join(out_path, model, dir_str, model_run_str), exist_ok=True)
+    file_str = f"ptype_predictions_{model}{model_run_str}z_fh{forecast_hour:02}.nc"
+    full_path = os.path.join(out_path, model, dir_str, model_run_str, file_str)
+    encoding_vars = [v for v in list(dataset.data_vars)]
+    encoding = {var: {"zlib": True, "complevel": 4, "least_significant_digit": 4} for var in encoding_vars}
+    dataset.to_netcdf(full_path, encoding=encoding)
```

Comment:
Can you add a to_zarr option here as well? The Vaisala team was interested in Zarr output since they operate on AWS and it would be easier for them to download.